runtime/app/executable-launcher.js

```diff
@@ -36,20 +36,28 @@ function launchExecutable (appDir, bridge) {
 
       var cp = childProcess.spawn(executablePath, [], {
         cwd: appDir,
-        stdio: 'inherit'
+        stdio: [ 'ignore', 'ignore', 'inherit' ]
       })
 
       cp.once('error', function onError (err) {
         bridge.logger.error(`Process(${cp.pid}) Unexpected error on child process '${appDir}'`, err.message, err.stack)
         cp.kill(/** SIGKILL */9)
       })
       cp.once('exit', (code, signal) => {
-        bridge.logger.info(`Process(${cp.pid}) exited with code ${code}, signal ${signal}, disconnected? ${!cp.connected}`)
-        bridge.exit(code, signal)
+        bridge.logger.info(`Process(${cp.pid}) exited with code ${code}, signal ${signal}`)
+        bridge.didExit(code, signal)
       })
-      bridge.implement((force) => {
-        bridge.logger.info(`Process(${cp.pid}) end of life, killing process.`)
-        cp.kill(force ? 'SIGKILL' : 'SIGTERM')
+      bridge.implement({
+        anrEnabled: true,
```

Comment:
Could this be configurable by system config?

Reply:
I don't think ANR should be configurable.